turbo.json

```diff
@@ -31,31 +38,47 @@
 				".custom-out-dir/types/**",
 				".svelte-kit/tsconfig.json"
 			],
-			"outputs": []
+			"outputs": [],
+			"outputMode": "new-only"
+		},
+		"format": {
+			"outputs": [
+				"src/**",
+				"scripts/**",
+				"shared/**",
+				"templates/**",
+				"types/**"
+			],
+			"outputMode": "new-only"
 		},
-		"format": {},
 		"prepare": {
 			"inputs": ["src/**", "source/**"],
 			"outputs": [
 				".svelte-kit/types/**",
 				".svelte-kit/tsconfig.json",
 				".custom-out-dir/types/**",
 				".custom-out-dir/tsconfig.json"
-			]
+			],
+			"outputMode": "new-only"
 		},
 		"test": {
 			"dependsOn": ["^build", "$CI", "$TURBO_CACHE_KEY"],
-			"outputs": []
+			"outputs": ["coverage/", "test-results/**"],
```

Comment:
If Turbo is saying that a test is failing I'd want to be able to find the `test-results` for it. I could probably find it by going back far enough in the CI history, but I'd rather make things easier for us. I don't think we're in any danger of using up our cache space?

`coverage` isn't being used at the moment because that's from the `adapter-node` output and those tests are currently disabled - though we should work to re-enable them. Is there any downside to keeping the output?

Reply:
Fair enough, we can revisit if we do end up somehow using too much space. I don't think we should re-enable the existing adapter-node tests, they're disabled for a reason — we'd be better off creating a more comprehensive set of tests in the template app itself so that all adapters can use them